tests/themes/tests.lua

```diff
@@ -0,0 +1,18 @@
+--- Simple script to load/test a theme.
+-- This is used from tests/themes/run.sh.
+
+local awful = require("awful")
+
+local steps = {
+    function(count)
+        if count <= 2 then
+            awful.spawn("xterm")
+        elseif #client.get() == 2 then
+            return true
```

Comment:
looks good however i am not sure about assertion criteria -- it will test what the theme hasn't paniced, but i am wondering mb it could make sense to also observe stderr for errors/warnings? or parsing text output sounds too much fragile for the tests?